.github/release-drafter.yml

```
@@ -44,5 +44,7 @@ autolabeler:
 
 template: |
   ## What Changed 👀
```

```suggestion
  ## 👀 What Changed
```
Probably also worth fixing out of consistency with the other headings.